commit f781bb84ee9c6323a1cbc6f178a82ce8ed9f965b (refs/changes/66/2666/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-09-27T14:09:54-07:00 (2 years ago)
    
    FWAPI-294 sdc-fwapi 'make docs' broken because fwrule doesn't publish its docs/
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/.npmignore b/.npmignore
index f0fbb1e..eadfdbb 100644
--- a/.npmignore
+++ b/.npmignore
@@ -3,7 +3,6 @@
 /Makefile
 /coverage
 /deps
-/docs
 /etc
 /src
 /test
diff --git a/package.json b/package.json
index 47bc495..5f2aedb 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "fwrule",
   "description": "Parser for Joyent firewall rules",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "author": "Joyent (joyent.com)",
   "keywords": [ "smartos", "triton" ],
   "repository": {
